internal/characters/yoimiya/abil.go

```diff
@@ -27,15 +27,26 @@ func (c *char) Attack(p map[string]int) (int, int) {
 	}
 
 	for i, mult := range attack[c.NormalCounter] {
-		ai.Mult = mult[c.TalentLvlAttack()]
-		totalMV += mult[c.TalentLvlAttack()]
+		//infusion to normal attack only
+		if c.Core.Status.Duration("yoimiyaskill") > 0 {
```

Comment:
L31 to L36 should stay in the snapshot function for readability. There's no real advantage/reason to split up the code here between Snapshot and Attack (unlike some other characters like Raiden which has a lot more complexity)

Reply:
@srliao Sounds good, will revert. I'm going to add a condition to the snapshot to prevent a bug where multiple calls to snapshot for a single attack was increasing the talent multiplier of the attack unexpectedly: https://discord.com/channels/845087716541595668/884485336657444894/962749496180244570